src/panels/lovelace/strategies/home/home-other-devices-view-strategy.ts

```diff
@@ -0,0 +1,208 @@
+    hass: HomeAssistant
+  ): Promise<LovelaceViewConfig> {
+    const allEntities = Object.keys(hass.states);
+
+    const otherDevicesFilters = OTHER_DEVICES_FILTERS.map((filter) =>
+      generateEntityFilter(hass, filter)
+    );
+
+    const otherDevicesEntities = findEntities(allEntities, otherDevicesFilters);
+
+    const sections: LovelaceSectionRawConfig[] = [];
+
+    const entitiesByDevice: Record<string, string[]> = {};
+    const entitiesWithoutDevices: string[] = [];
+    for (const entityId of otherDevicesEntities) {
+      const stateObj = hass.states[entityId];
+      if (!stateObj) continue;
+      const { device } = getEntityContext(
+        stateObj,
+        hass.entities,
+        hass.devices,
+        hass.areas,
+        hass.floors
+      );
+      if (!device) {
+        entitiesWithoutDevices.push(entityId);
+        continue;
+      }
+      if (!(device.id in entitiesByDevice)) {
+        entitiesByDevice[device.id] = [];
+      }
+      entitiesByDevice[device.id].push(entityId);
+    }
+
+    const devicesEntities = Object.entries(entitiesByDevice).map(
+      ([deviceId, entities]) => ({
+        device_id: deviceId,
+        entities: entities,
+      })
+    );
+
+    const helpersEntities = entitiesWithoutDevices.filter((entityId) => {
+      const domain = computeDomain(entityId);
+      return isHelperDomain(domain);
+    });
+
+    const otherEntities = entitiesWithoutDevices.filter((entityId) => {
+      const domain = computeDomain(entityId);
+      return !isHelperDomain(domain);
+    });
+
+    const batteryFilter = generateEntityFilter(hass, {
+      domain: "sensor",
+      device_class: "battery",
+    });
+
+    const energyFilter = generateEntityFilter(hass, {
+      domain: "sensor",
+      device_class: ["energy", "power"],
+    });
+
+    const primaryFilter = generateEntityFilter(hass, {
+      entity_category: "none",
+    });
+
+    for (const deviceEntities of devicesEntities) {
+      if (deviceEntities.entities.length === 0) continue;
+
+      const batteryEntities = deviceEntities.entities.filter((e) =>
+        batteryFilter(e)
+      );
+      const entities = deviceEntities.entities.filter(
+        (e) => !batteryFilter(e) && !energyFilter(e) && primaryFilter(e)
+      );
+
+      if (entities.length === 0) {
+        continue;
+      }
+
+      const deviceId = deviceEntities.device_id;
+      const device = hass.devices[deviceId];
+      let heading = "";
+      if (device) {
+        heading =
+          computeDeviceName(device) ||
+          hass.localize("ui.panel.lovelace.strategy.home.unamed_device");
+      }
+
+      sections.push({
+        type: "grid",
+        cards: [
+          {
+            type: "heading",
+            heading: heading,
+            tap_action: device
+              ? {
+                  action: "navigate",
+                  navigation_path: `/config/devices/device/${device.id}`,
+                }
+              : undefined,
```

Comment:
```suggestion
              : { action: "none" },
```

undefined is handled as a more-info action.